code/modules/research/fabricators.dm

```diff
@@ -81,7 +82,7 @@
 
 /obj/machinery/r_n_d/fabricator/emag()
 	sleep()
-	if(!research_flags &ACCESS_EMAG)
```

Comment:
Does this even work? It seems like the check will just do `0 & ACCESS_EMAG` which is always false.